packages/labs/analyzer/src/lib/javascript/classes.ts

```diff
@@ -135,7 +150,7 @@ const getClassDeclarationName = (declaration: ts.ClassDeclaration) => {
   if (name === undefined) {
     throw new DiagnosticsError(
       declaration,
-      'Unexpected class declaration without a name'
+      'Internal error: expected class declaration to either have a name or be a default export'
```

Comment:
I don't think an error should both be a diagnostic and internal. If this is a diagnostic, can we handle it instead of throwing? If we throw, where is this caught and recovered from?